include/utils.hpp

```diff
@@ -248,8 +248,7 @@ constexpr auto toHex(size_t aByte)
  * @param[out] vpdVector - VPD in vector form.
  * @param[in] vpdStartOffset - Offset of VPD data in EEPROM.
  */
-void getVpdDataInVector(std::fstream& vpdFileStream,
-                        const std::string& vpdFilePath,
+void getVpdDataInVector(const std::string& vpdFilePath,
```

Comment:
Remove oxygen "@param[in] vpdFileStream"

Reply:
Done.